grain/oss/build_whl.sh

```diff
@@ -126,6 +126,16 @@ main() {
   $PYTHON_BIN -m pip install --find-links="${OUTPUT_DIR}/all_dist" --pre "${PKG_NAME}"
   $PYTHON_BIN -m pip install jax
   $PYTHON_BIN grain/_src/core/smoke_test_with_jax.py
+  $PYTHON_BIN grain/_src/core/tree_lib_jax_test.py
```

Comment:
having few tests run differently from the rest is a bit brittle imo. Is there a way for us to run a test suite with something like pytest? or unittest discover?

Reply:
yeah, make sense, thanks. I think ideally we would run the whole test suit together in the long term

Reply:
Ok, I managed to migrate these few tests to `pytest --pyargs` call and they are discoverable. I tried running the whole suite but I get about 20 errors (mostly because I couldn't pass abseil flags from pytest call). I added a todo for that, as I think this is a separate thing to work on. 

Ideally I would prefer to first merge https://github.com/google/grain/pull/954 so I don't need to repeat the part fixing Windows tests.